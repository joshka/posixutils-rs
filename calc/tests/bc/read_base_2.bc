ibase = 2
0
1
0110110100001
1011110.0001001
quit
