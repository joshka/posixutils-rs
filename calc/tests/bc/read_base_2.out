0
1
3489
94.0703125
